test/sagemaker_tests/pytorch/inference/integration/sagemaker/test_mnist.py

```diff
@@ -118,3 +124,50 @@ def _test_mnist_distributed(
         output = predictor.predict(data)
 
         assert output.shape == (batch_size, 10)
+
+        #  Check for Cloudwatch logs
+        if verify_logs:
+            _check_for_cloudwatch_logs(endpoint_name)
+
+def _check_for_cloudwatch_logs(endpoint_name):
+    client=boto3.client('logs')
+    identifyLogStream = client.describe_log_streams(
+        logGroupName='/aws/sagemaker/Endpoints/'+endpoint_name,    
+        orderBy='LastEventTime',
+        descending=True,
+        limit=5
+    )
+
+    indexError=False;
+    try:
```

Comment:
why suppress the indexerror exception? You can remove the true/false logic and do something like this:

```
   try:
        logStreamName=identifyLogStream['logStreams'][0]['logStreamName']        
    except IndexError as e:
        raise RuntimeError(f"Unable to look up log streams in log stream {some_variable_that_identifies_log_stream_name}") from e
```

Reply:
Thanks for the suggestion. This has been implemented.